### ISSUE TITLE
Feature Request: Stealth 2.0

### ISSUE THREAD

**Issue by Turtlelord26** [trowl223/Compound_X#191]
Currently we have stealth rules that only partially exist, particularly as concerns dedicated stealth classes.

Making a sci-fi set game is a wonderful opportunity to build a stealth system with more complexity than "can you roll your disgustingly min-maxed stealth check higher than all of these plebs' awareness checks?". At the same time our aesthetic suggests we avoid adding any more complexity than we have to. 

My thought is a system that mechanically distinguishes between I-am-quiet stealth and invisibility-cloak-tech stealth, where you awareness check at mundane stealth but really should have tech detection equipment for tech stealth. You can probably detect stealthies with the wrong equipment but it's definitely harder.

**Comment by 1sourcecontrol** [comment]
That sounds like a cool idea! 

I'm all for interesting differences in kind. I think if we were to implement this, we'd have to nerf cloaking into the ground (1 turn usage with a long recharge, or something of that sort). Otherwise a player with cloaking will be unstoppable in a lot of scenarios (I don't like designing on the hope that the DM will always build into their campaign ways to stop every character class). BTW I wouldn't mind pursuing a cloaking redesign/quasi-nerf.

**Comment by Turtlelord26** [comment]
I'm wary of that kind of approach. Adding counterplay is going to be more fun than nerfing stealth that far into the ground up front. Keep in mind we're currently leaning toward abolishing the feat that lets non-stealth classes use tech stealth.

We can help the DM design counterplay into every scenario they come up with. If we can 
-tie at least basic tech detection into relatively mundane base equipment (think like targeting scanners can incidentally detect stealth with an aware operator), 
-include some traps whose triggers don't care how visible you are, 
-explicitly add scent capabilities to some of our crittery monsters, 
-etc.
then it won't be terribly challenging for a DM to find counterplay options when they have a stealth PC in their party. And in theory by the time anyone's DMing a higher level party they'll be more experienced at coming up with creative challenges for their particular playgroup.

**Comment by 1sourcecontrol** [comment]
>more fun than nerfing stealth that far into the ground up front

What I meant was, the cloaking turn would be nerfed but counterbalanced by the large jump in power. At level 1, having 1 turn of cloak with a 2 turn/1 turn recharge would be pretty reasonable. Especially because of the incredible buff to cloaking power for not having to use stealth checks. 

*edit*: @bleehu can speak to this more than I can, because I wasn't there; I want to avoid what happened in the early cloaking days, where cloaking was so strong that players were nigh' undetectable. Putting in a short cloak duration would be a very easy way to guarantee some balance.

**Comment by Turtlelord26** [comment]
Heavens, no, I never meant to imply tech stealth would be free from stealth checks. Technology builds on and enhances existing capabilities - you could have the most decked out and gadgeted bow in the history of mankind but if you don't know how to shoot you aren't reliably going to hit anything. Rodger the Arena Fighting Robot is **never** going to be able to use tech stealth effectively.

Revisit the last sentence of the first post.

**Comment by 1sourcecontrol** [comment]
Well, Rodger does have a higher stealth skill than my paladin Andreas......

I noticed that the name of this issue seems to be different than the ensuing conversation; although, I can't tell for sure if "stealth rules that don't exist" are talking about active cloaking or actual stealth rules. For reference, I significantly altered cloaking rules in the C2.0 merge.

**Comment by Turtlelord26** [comment]
A valid point. The issue results from the Chasing Kalihan run two weeks ago, where I was only looking at the play branch (and hence not c2.0). Name updated to reflect such.

**Comment by bleehu** [comment]
@Turtlelord26, has this been adequately answered in the combat 2.0 stealth rules in RC?

**Comment by Turtlelord26** [comment]
@bleehu No. We do have what looks like functional stealth now, which is good, but I still want to propose an additional feature or two. It hasn't been addressed yet due to the low priority assigned it at the last dev meeting.
